src/cli/commands/private-cmds/cat-object-cmd.ts

```diff
@@ -9,7 +9,7 @@ export default class CatObject implements LegacyCommand {
   opts = [
```

Comment:
these changes are from another PR that's already merged to master... Due to overzealous re-basing on my part